### PR TITLE
docs(PPDSC-2477): remove sourcepoint component

### PR DESCRIPTION
**Pull request by mutebg**
PPDSC-2477

**What**

What did you do - removed the Consent component, the consent code will be loaded directly from tealium

**I have done:**
- [ ] Written unit tests against changes
- [ ] Written functional tests against the component and/or NewsKit site
- [ ] Updated relevant documentation

**I have tested manually:**
- [ ] The feature's functionality is working as expected on Chrome, Firefox, Safari and Edge
- [ ] The screen reader reads and flows through the elements as expected.
- [ ] There are no new errors in the browser console coming from this PR.
- [ ] When visual test is not added, it renders correctly on different browsers and mobile viewports (Safari, Firefox, small mobile viewport, tablet)
- [ ] The Playground feature is working as expected

**Before:**

**After:**

**Who should review this PR:**

**How to test:**

### CODE REVIEW

**Comment by newskit-engineering** [comment]
You can preview these changes on:
  Documentation Site: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2477-consolidate-sourcepoint
  Storybook: http://ncu-newskit-docs.s3-website-eu-west-1.amazonaws.com/ppdsc-2477-consolidate-sourcepoint/storybook

File: site/pages/_document.tsx [hunk 1]
```diff
@@ -246,14 +236,14 @@ export default class MyDocument extends Document {
               }
             `}
           />
-          {isSiteEnvProduction && (
```
Comment (mstuartf) [inline site/pages/_document.tsx:249]: Why was this condition there originally?

Comment (mutebg) [reply site/pages/_document.tsx:249]: @Vanals I see this has been added to remove it for SVG Previewer, do we still need that?

Comment (mstuartf) [reply site/pages/_document.tsx:249]: This rings a bell. I think it breaks a figma plugin or something?

Comment (Vanals) [reply site/pages/_document.tsx:249]: hey! sorry saw this randomly now.
I think the reason was that tealium did break our website from being rendered in a Iframe. This stopped designers from being able to use the previewer. Removing it from them allows them to use the Previewers with no issue, so yeah this should stay I think

Comment (mutebg) [reply site/pages/_document.tsx:272]: What if we move that to _app.tsx and from there we can have access to url "path" and disable it only for preview URL?

Comment (mutebg) [reply site/pages/_document.tsx:272]: Reverted the code

**Review by JohnTParsons — DISMISSED** [review]

**Review by JohnTParsons — APPROVED** [review]

**Pull request merged**